modules/codemod/lib/v9/spec/deprecateStack.spec.ts

```diff
@@ -361,7 +361,7 @@ describe('Canvas Kit Deprecate Stack Codemod', () => {
       expectTransform(input, expected);
     });
 
-    it('should properly transform the spacing prop on specific components that extend Stack', () => {
+    it('should properly transform the spacing prop on the ActionBar component that extends Stack', () => {
```

Comment:
This test makes more sense to me since `ActionBar.List` _does_ extend `HStack`. However, the `ActionBar` container component doesn't extend `Stack` so it shouldn't have a `spacing` in the test input (or a `gap` in the test output).

Reply:
Updated